storage/postgres/postgres.go

```diff
@@ -108,6 +108,24 @@ func (p *Postgres) Close(ctx context.Context) error {
 	return nil
 }
 
+func (p *Postgres) ListObjectsByType(
+	ctx context.Context,
+	filter storage.ListObjectsFilter,
+) (storage.ObjectIterator, error) {
+	ctx, span := p.tracer.Start(ctx, "postgres.ListObjectsByType")
+	defer span.End()
+
+	sql := "SELECT DISTINCT object_type, object_id FROM tuple WHERE store='%v' AND object_type='%v'"
+	args := []interface{}{filter.StoreID, filter.ObjectType}
+
+	rows, err := p.pool.Query(ctx, fmt.Sprintf(sql, args...))
```

Comment:
I think the use of fmt.Sprintf means that this may be vulnerable to injections. You can simple use positional arts here $1, $2, to remove that use.